packages.bib

```diff
@@ -25,23 +25,22 @@ @Manual{R-epidatasets
 
 @Manual{R-epidatr,
   title = {epidatr: Client for Delphi's Epidata API},
-  author = {Jacob Bien and Logan Brooks and David Farrow and Pedrito Maynard-Zhang and Alex Reinhart and Ryan Tibshirani and Samuel Gratzl},
-  note = {R package version 0.6.0},
-  url = {https://github.com/cmu-delphi/epidatr},
+  author = {Logan Brooks and Dmitry Shemetov and Samuel Gratzl},
   year = {2023},
+  note = {https://cmu-delphi.github.io/epidatr/, https://cmu-delphi.github.io/delphi-epidata/},
 }
 
 @Manual{R-epipredict,
   title = {epipredict: Basic epidemiology forecasting methods},
-  author = {Daniel McDonald and Ryan Tibshirani and Logan Brooks and Rachel Lobay and Maggie Liu and Ken Mawer and Chloe You},
+  author = {Daniel McDonald and Ryan Tibshirani and Logan Brooks and Rachel Lobay},
```

Comment:
We may want to rethink author credit in both of our packages. These both strike me as overly exclusive.